src/views/Main/Visualizations/GeoVisualization.js

```diff
@@ -24,8 +24,8 @@ export class GeoVisualization extends React.Component {
     let minLong = 180;
 
     const markers = data.map((point) => {
-      const lat = point[latName][0];
-      const long = point[longName][0];
+      const lat = parseFloat(point[latName][0]);
```

Comment:
you have to check the result. if the input is invalid, this returns `NaN`, which you don't want. check all of these.